tuist/tuist
# Add support for watchOS apps

### Short description 📝

Adds support for watchOS apps.

### Solution 📦

Add support for watchOS / watchOS Extension products as well as appropriate dependency rules.

### Implementation 👩‍💻👨‍💻

- [x] Add product types
- [x] Add fixture & acceptance test
- [x] Update graph / link rules
- [x] Add embed watch build phase
- [x] Add watch build settings

### Test Plan 🛠

- Run `tuist generate` within `fixtures/ios_app_with_watchapp2`
- Verify the generated project contains a Watch App & Watch App Extension
- Compare against a manually created project with a Watch App

### Notes

For follow up PRs:

- [ ] Update schemes to include a runnable watch apps (requires using `RemoteRunnable` launch actions)
- [ ] Add support for watch app containers (aka standalone watch apps)
